client/app/hosting/configuration/CONFIGURATION.html

```diff
@@ -0,0 +1,49 @@
+<div class="container-fluid px-0" data-ng-controller="HostingTabConfigurationCtrl as ctrl">
+    <div class="row">
+        <div class="col-md-9">
+            <div data-ovh-alert="{{ alerts.main }}"></div>
+
+            <oui-spinner data-size="l" data-ng-if="ctrl.loading.init"></oui-spinner>
+            <div data-ng-if="!ctrl.loading.init && ctrl.hasResult">
+                <div data-ng-include="ctrl.templates.runtimeList"></div>
+                <br><br>
+
+                <div data-ng-include="ctrl.templates.envvarList"></div>
+            </div>
+
+            <div data-ng-if="!ctrl.loading.init && !ctrl.hasResult">
+                <div class="alert alert-info mt-5" role="alert">
+                    <div data-i18n-static="hosting_tab_CONFIGURATION_empty"></div>
+                </div>
+            </div>
+        </div>
+
+        <aside class="col-md-3 mt-5 mt-lg-0">
+            <div class="mb-5">
+                <button type="button" class="btn btn-block btn-default text-wrap" title="{{i18n.hosting_tab_CONFIGURATION_runtime_add}}"
+                        data-i18n-static="hosting_tab_CONFIGURATION_runtime_add"
+                        data-ng-click="setAction('configuration/runtime/add-or-edit/hosting-configuration-runtime-add-or-edit', {})"
+                        ng-if="ctrl.canAddRuntime()">
+                </button>
+
+                <button type="button" class="btn btn-block btn-default text-wrap" title="{{i18n.hosting_tab_CONFIGURATION_runtime_quota_reached}}"
+                        data-i18n-static="hosting_tab_CONFIGURATION_runtime_add"
+                        ng-if="!ctrl.canAddRuntime()"
+                        disabled>
+                </button>
+
+                <button type="button" class="btn btn-block btn-default text-wrap" title="{{i18n.hosting_tab_CONFIGURATION_envvar_add}}"
+                        data-i18n-static="hosting_tab_CONFIGURATION_envvar_add"
+                        data-ng-click="setAction('configuration/envvar/add-or-edit/hosting-configuration-envvar-add-or-edit', {})"
+                        ng-if="ctrl.canAddEnvvar()">
+                </button>
+
+                <button type="button" class="btn btn-block btn-default text-wrap" title="{{i18n.hosting_tab_CONFIGURATION_envvar_quota_reached}}"
+                        data-i18n-static="hosting_tab_CONFIGURATION_envvar_add"
+                        ng-if="!ctrl.canAddEnvvar()"
+                        disabled>
+                </button>
```

Comment:
There should not be 4 button elements for only two buttons.
Use the condition inside the `data-ng-disabled` attribute like so:
```html
<button type="button" class="btn btn-block btn-default text-wrap"
        title="{{ctrl.canAddEnvvar() ?
                 i18n.hosting_tab_CONFIGURATION_envvar_add :
                 i18n.hosting_tab_CONFIGURATION_envvar_quota_reached}}"
        data-i18n-static="hosting_tab_CONFIGURATION_envvar_add"
        data-ng-disabled="ctrl.canAddEnvvar()">
</button>
```
preferably using a variable instead of a function to prevent angular from calling it repeatedly.

Reply:
Basically you just replace the two similar buttons (one active, one disabled) with what I gave you.